content/en/docs/components/katib/overview.md

```diff
@@ -5,165 +5,166 @@ weight = 10
                     
 +++
 
-{{% beta-status 
+{{% beta-status
   feedbacklink="https://github.com/kubeflow/katib/issues" %}}
 
-Use Katib for automated tuning of your machine learning (ML) model's 
-hyperparameters and architecture.
+Katib is a Kubernetes-native project for automated machine learning (AutoML).
+For example, to run hyperparameter tuning or neural architecture search.
+Learn more about AutoML at [fast.ai](https://www.fast.ai/2018/07/16/auto-ml2/),
+[Google Cloud](https://cloud.google.com/automl),
+[Microsoft Azure](https://docs.microsoft.com/en-us/azure/machine-learning/concept-automated-ml#automl-in-azure-machine-learning) or
+[Amazon SageMaker](https://aws.amazon.com/blogs/aws/amazon-sagemaker-autopilot-fully-managed-automatic-machine-learning/).
+
+Katib is the framework agnostic to machine learning frameworks.
+It can tune hyperparameters of applications written in any language
+of the users' choice and natively supports many machine learning frameworks,
+such as TensorFlow, MXNet, PyTorch, XGBoost, and others.
+
+The [Katib project](https://github.com/kubeflow/katib) is open source.
+The [developer guide](https://github.com/kubeflow/katib/blob/master/docs/developer-guide.md)
+is a good starting point for developers who want to contribute to the project.
```

Comment:
Can you please use my previous suggestion (old text from the existing Markdown) and throw in the cloud links, so we don't reinvent the wheel? There's quite a lot of review in this PR :)

Reply:
@8bitmp3 Sure, I've made section as you said:
https://github.com/kubeflow/website/blame/b946be02fb367f8e9c2fb5d88a7818ddd7c18ee8/content/en/docs/components/katib/overview.md#L11-L31